plugin.video.ifttt/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+	id="plugin.video.ifttt"
+	name="IFTTT Service"
+	version="1.0.0"
+	provider-name="plucka"
+	>
+	<requires>
+		<import addon="xbmc.python" version="2.26.0"/>
+	</requires>
+	<extension point="xbmc.service" library="default.py" start="login"/>
```

Comment:
your main extension point is a service and not a script, so this addon is not a plugin. Keep the service one.